docs/sources/guides/whats-new-in-v6-5.md

```diff
@@ -35,6 +38,65 @@ Grafana 6.5 comes with a lot of new features and enhancements.
 
 More details of above and highlights will be added as we're getting closer to the stable release.
 
+### CloudWatch data source improvements
+
+In this release, several feature improvements and additions were made in the CloudWatch data source. This work has been done in collaboration with the Amazon CloudWatch team.
+
+#### GetMetricData API
+
+For Grafana version 6.5 or higher, all API requests to GetMetricStatistics have been replaced with calls to GetMetricData, following Amazon’s [best practice to use the GetMetricData API](https://aws.amazon.com/premiumsupport/knowledge-center/cloudwatch-getmetricdata-api) instead of GetMetricStatistics, because data can be retrieved faster at scale with GetMetricData. This change provides better support for CloudWatch metric math and enables the use of automatic search expressions.
+
+While GetMetricStatistics qualified for the CloudWatch API free tier, this is not the case for GetMetricData calls. For more information, please refer to the [CloudWatch pricing page](https://aws.amazon.com/cloudwatch/pricing/).
+
+#### Dynamic queries using dimension wildcards
```

Comment:
This is a big new feature - I would add a sentence describing the impact of dynamic queries as the second sentence. You have something describing the impact but it is in the middle of a block of text further down and there is a risk that the reader might skip over it.

> For example, when new instances get created as part of an auto scaling event, they will automatically appear in the graph without you having to track the new instance IDs.